### PR TITLE
Adding github actions

### PR DESCRIPTION
**Pull request by scalvert**

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: CI Build
+
+on:
+  push:
+    branches:
+      - master
```
Comment (renatoi) [inline .github/workflows/ci-build.yml:6]: The name of the `master` branch` is `main`.

Comment (scalvert) [reply .github/workflows/ci-build.yml:6]: Blarg

Comment (scalvert) [reply .github/workflows/ci-build.yml]: Fixed.

**Comment by renatoi** [comment]
:shipit:

**Pull request merged**